generated/beats/fields.ecs.yml

```diff
@@ -2505,877 +3320,4822 @@
+      description: TODO
+      default_field: false
+    - name: defense_evasions.total_memory_size
+      level: custom
+      type: keyword
+      ignore_above: 1024
+      description: TODO
+      default_field: false
+    - name: domain
+      level: custom
+      type: keyword
+      ignore_above: 1024
+      description: TODO
+      default_field: false
+    - name: env_variables
+      level: custom
+      type: keyword
+      ignore_above: 1024
+      description: TODO
+      default_field: false
+    - name: executable
+      level: extended
+      type: keyword
+      ignore_above: 1024
+      multi_fields:
+      - name: text
+        type: text
+        norms: false
+        default_field: false
+      description: Absolute path to the process executable.
+      example: /usr/bin/ssh
+    - name: exit_code
+      level: extended
+      type: long
+      description: 'The exit code of the process, if this is a termination event.
+
+        The field should be absent if there is no exit code for the event (e.g. process
+        start).'
+      example: 137
+      default_field: false
+    - name: gid
+      level: custom
+      type: long
+      description: TODO
+      default_field: false
+    - name: group
+      level: custom
+      type: keyword
+      ignore_above: 1024
+      description: TODO
+      default_field: false
+    - name: handle
+      level: custom
+      type: nested
+      description: TODO
+      default_field: false
+    - name: handle.handle_id
+      level: custom
+      type: long
+      description: TODO
+      default_field: false
+    - name: handle.handle_name
+      level: custom
+      type: keyword
+      ignore_above: 1024
+      description: TODO
+      default_field: false
+    - name: handle.handle_type
+      level: custom
+      type: keyword
+      ignore_above: 1024
+      description: TODO
+      default_field: false
+    - name: has_unbacked_execute_memory
+      level: custom
+      type: boolean
+      description: TODO
+      default_field: false
+    - name: hash.imphash
+      level: extended
+      type: keyword
+      ignore_above: 1024
+      description: Imphash.
+      default_field: false
+    - name: hash.md5
+      level: extended
+      type: keyword
+      ignore_above: 1024
+      description: MD5 hash.
+    - name: hash.sha1
+      level: extended
+      type: keyword
+      ignore_above: 1024
+      description: SHA1 hash.
+    - name: hash.sha256
+      level: extended
+      type: keyword
+      ignore_above: 1024
+      description: SHA256 hash.
+    - name: hash.sha512
+      level: extended
+      type: keyword
+      ignore_above: 1024
+      description: SHA512 hash.
+    - name: hash_matched_module
+      level: custom
+      type: boolean
+      description: TODO
+      default_field: false
+    - name: imphash
+      level: custom
+      type: keyword
+      ignore_above: 1024
+      description: TODO
+      default_field: false
+    - name: is_sensor
```

Comment:
Consider changing to `is_endpoint`. Sensor is an SMP term not an Elastic Endpoint term.